packages/vite/src/node/ssr/ssrExternal.ts

```diff
@@ -40,11 +40,15 @@ export function createIsConfiguredAsSsrExternal(
     typeof noExternal !== 'boolean' &&
     createFilter(undefined, noExternal, { resolve: false })
 
+  const targetConditions =
+    config.ssr.resolve?.externalConditions || config.resolve.conditions
```

Comment:
Should we prefer ssr.resolve.conditions here? E.g.

`config.ssr.resolve?.externalConditions || config.ssr.resolve?.conditions || config.resolve.conditions`

Reply:
Makes sense - I played it safe by falling back to prior behavior, but I'll adjust it to your rec 👍.